kafka/include/userver/kafka/topic.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+USERVER_NAMESPACE_BEGIN
+
+namespace kafka {
+
+struct OffsetRange final {
+    std::int64_t high{};
```

Comment:
why `std::int64_t`, not `std::uint32_t`?
also add comments for fields and structure